tests/datasets/test_chesapeake.py

```diff
@@ -212,6 +212,9 @@ def test_plot(self, dataset: ChesapeakeCVPR) -> None:
         plt.close()
         dataset.plot(x, show_titles=False)
         plt.close()
-        x['prediction'] = x['mask'][:, :, 0].clone().unsqueeze(2)
+        if x['mask'].ndim == 2:
```

Comment:
Can you explain this change?

Reply:
Okay, so now we either load a H x W mask, or a C x H x W mask. @nilsleh was also asking for support for multi-channel masks in our trainer. There's an open TODO. We'll figure that out someday.